File: modules/Volo/modules/Volo.Chat/angular/projects/chat/src/lib/models/chat-contact-dto.ts
export class ChatContactDto {
  userId: string;
  name: string;
  surname: string;
  username: string;
  lastMessageSide: number;
  lastMessage: string;
  lastMessageDate: Date | string;
  unreadMessageCount: number;

  constructor(initialValues: Partial<ChatContactDto> = {}) {
    if (initialValues) {
      for (const key in initialValues) {
        if (initialValues.hasOwnProperty(key)) {
          this[key] = initialValues[key];
        }
      }
    }
  }
}

File: modules/Volo/modules/Volo.Chat/src/Volo.Chat.Application.Contracts/Volo/Chat/Users/ChatContactDto.cs
using System;
using Volo.Chat.Messages;

namespace Volo.Chat.Users
{
    public class ChatContactDto
    {
        public Guid UserId { get; set; }

        public string Name { get; set; }

        public string Surname { get; set; }

        public string Username { get; set; }

        public ChatMessageSide LastMessageSide { get; set; }

        public string LastMessage { get; set; }

        public DateTime? LastMessageDate { get; set; }

        public int UnreadMessageCount { get; set; }
    }
}